src/components/onboardingFormData/PersonalAndBillingDataSection.tsx

```diff
@@ -985,7 +985,7 @@ export default function PersonalAndBillingDataSection({
             </>
           )}
           {/* indirizzo mail di supporto */}
-          {!institutionAvoidGeotax && productId !== 'prod-interop' && (
```

Comment:
It's not correct, you do not have to show this input field for SCPs that are submitting the onboarding request to interop. The change shall not affect other institution types for this product.

See the mockup: https://www.figma.com/design/14WthFMcY9UU6G3BqnZHi2/Area-Riservata?node-id=34725-37109&t=Wx57V9C63HNH9628-4